pkgs/applications/science/logic/symbiyosys/default.nix

```diff
@@ -31,6 +31,9 @@ stdenv.mkDerivation {
 
     substituteInPlace sbysrc/sby_core.py \
       --replace '##yosys-program-prefix##' '"${yosys}/bin/"'
+
+    substituteInPlace sbysrc/sby.py \
```

Comment:
Can we use patchShebangs instead?

Reply:
Hm, not out of the box, because the source line is `/usr/bin/env python3` and nix's python3 package only has `python`, not `python3`, so the patching is a no-op. Is there an obvious way around that? I don't see any options for this in the patchShebangs source.

Reply:
I was just asking incase your forgot it.

Reply:
I definitely forgot about it! But lucky for me, it looks like I also can't use it :P